.circleci/config.yml

```diff
@@ -457,7 +457,7 @@ jobs:
               --git-version "${CIRCLE_TAG}" \
               --app-version "${CIRCLE_TAG}" \
               --circle-url "https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}" \
-              --key-file circle-sa-key.json \
+              --key-file ~/workbench/api/circle-sa-key.json \
```

Comment:
@aweng98 for another day, but I see a lot of duplication of the working dir `~/workbench`. Can that be a top-level parameter (or container-level)?

Also is there a more canonical location for some of these than the user's home directory? I have a hard time understanding who the user is with Docker, but that's probably just a lack of familiarity.